catalogue/webapp/pages/works.js

```diff
@@ -97,6 +98,34 @@ const Works = ({ works }: Props) => {
     );
   }
 
+  const workTypes = [
```

Comment:
As a point, are these not categories? A/V I don't think will ever be a `workType`.
Just asking for thinking about how we move forward with this.

Reply:
You're right – badly named variable on my part.